test/referral_chaser_test.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'test_helper'
+
+class GitHubLdapReferralChaserTestCases < GitHub::Ldap::Test
+
+  def setup
+    @mock_connection = GitHub::Ldap.new({
+        admin_user: "uid=admin,dc=github,dc=com",
+        admin_password: "passworD1",
+        port: 389
+    })
```

Comment:
Still need to use the `options` defined in `GitHub::Ldap::Test` for the right host.

Note, though, that these failures are because `capabilities` are being checked to aggressively, causing a query to occur when creating a new object, rather than lazily loading the capabilities as needed. This is out-of-scope from this PR, but it's a constraint to getting the build to pass now.